### DISCUSSION TITLE
Is there documentation on the event handlers?

### DISCUSSION THREAD

**Discussion by ENBYSS** [nuxt/nuxt#16182] [Questions]
So I've just now found out about **defineCachedEventHandler** and **defineLazyEventHandler**, and I couldn't manage to find any documentation regarding these. After some digging I believe that these _might_ internally use **nitro** - however I'm wondering if there's any concrete documentation regarding the usage of these two methods _(even if they redirect to documentation elsewhere)_?

**Comment by TheAlexLichter** [comment]
Yes, feel free to check out the [Nitro docs](https://nitro.unjs.io/guide/introduction/cache) regarding it ☺️.

**Reply by ENBYSS** [reply]
What about `defineLazyEventHandler` though?
